test/integration/cart_integration_test.rb

```diff
@@ -91,6 +91,30 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
   end
 
   test 'an unauthorized user cannot checkout until logged in' do
-    
+    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
+    visit "/items/#{item.id}"
+
+    click_link_or_button('Add to Cart')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.